components/scream/src/share/io/tests/io.cpp

```diff
@@ -374,7 +378,7 @@ ekat::ParameterList get_in_params(const std::string type,
       + "." + t_first_write.to_string() + ".nc";
 
   in_params.set<std::string>("Filename",filename);
-  in_params.set<vos_type>("Fields",{"field_1", "field_2", "field_3", "field_packed"});
+  in_params.set<vos_type>("Fields Names",{"field_1", "field_2", "field_3", "field_packed"});
```

Comment:
This might be too much of a pain, but what do you think about `"Field Names"` instead of `"Fields Names"`?

Reply:
Not a pain at all. I actually wondered quite a bit what was the correct English formulation between the two. If you say `Field Names` is the right one, I can switch them all.